lib/metrics.js

```diff
@@ -166,6 +166,12 @@ Histogram.prototype.flush = function () {
       return a - b;
     };
     this.samples.sort(numericalSortAscending);
+
+    if (this.aggregates.includes('median')) {
```

Comment:
`includes()` has been around for a while, so I figured it’s OK to use (and I’m happy to do a follow-up PR or another commit here to modernize the rest of the instead of using `indexOf()`). If not, I’ll flip this to using `indexOf()` instead.

Reply:
This seems fine considering other syntax we're using in this package.